src/read_statmech_quantities.jl

```diff
@@ -21,33 +51,27 @@ end
 """
     setup_equilibrium_constants()
 
-Returns a Dict holding the default (Barklem & Collet 2016) log equilibrium constants.
+Returns a Dict holding the default (Barklem & Collet 2016) log equilibrium constants, which are in
+pressure units.
 
-There are two things of note about about equation 7 in this paper.  The first is that it defines K 
-as the reciprocal of what is actually supplied in the data table.  The second is that in order for 
-it to be correct, m must be interpreted as a reduced mass, m₁m₂/(m₁ + m₂).  This can be verified by 
+In equation 7 in this paper, m is the reduced mass, m₁m₂/(m₁ + m₂).  This can be verified by 
 re-deriving the equilibrium constants from the partition functions and dissolution constants 
 provided by the paper.
 """
 setup_equilibrium_constants() = read_partition_funcs(
-                                 joinpath(_data_dir, "BarklemCollet2016-equilibrium_constants.dat"))
+                                 joinpath(_data_dir, "BarklemCollet2016-equilibrium_constants.dat"),
+                                 transform=x->x+1) #convert from log(mks) to log(cgs)
 
 """
-    functon setup_atomic_partition_funcs()
+    functon read_atomic_partition_funcs([transform=identity])
 
-Constructs a Dict holding functions for various atomic species that computes the partition function
-at an input temperature.
+Constructs a Dict holding tables containing partition function or equilibrium constant values across
+temperature.
 
-In more detail, the partition function is linearly interpolated from values tabulated from Barklem 
-& Collet (2016). The partition functions can only be evaluated at temperatures between 1e-5 K and 
-1e4 K. The tabulated values were NOT computed with a cutoff principle quantum number. This can lead
-to a couple percent error for some neutral species in solar type stars (I think the problem becomes
-worse in hotter stars). See the paper for more details.
+The optionsl argument, `transform` is applied to each value. It is used to do unit conversions, etc.
```

Comment:
```suggestion
The optional argument, `transform` is applied to each value. It is used to perform unit conversions,
when loading equilibrium constants
```
I fixed a typo and added a clause to clarify the usage of the keyword (I was originally very confused since partition functions don't have units)